e2e/alphabet-card-detail.test.ts

```diff
@@ -1,51 +1,120 @@
 import { by, device, element, expect } from 'detox';
 
-describe('Detail', () => {
-    beforeAll(async () => {
-        await device.launchApp();
-    });
+interface ITappableAsync {
+    tap(): Promise<void>;
+}
+
+const multiTap = async (tappable: ITappableAsync, numberOfTaps: number): Promise<void> => {
+    for (let _i of Array(numberOfTaps)
+        .fill(0)
+        .map((_, index) => index)) {
+        await tappable.tap();
+    }
+};
 
+describe('Detail', () => {
     beforeEach(async () => {
+        await device.launchApp({
+            launchArgs: {
+                configOverrides: {
+                    config: {
+                        env: {
+                            TARGET_ALPHABET_NAME: `alphabet-english`,
+                            BASE_API_URL: `http://10.0.2.2:3131/api`,
```

Comment:
nice. I wonder if we want to have a test.env type file for this. The url may be different on my machine or in the CI, for example. Add a story for this.

Reply:
Story is [here](https://www.pivotaltracker.com/story/show/188014886).